core/app/models/spree/page_section.rb

```diff
@@ -187,7 +187,7 @@ def dup
     end
 
     def restore_design_settings_to_defaults
```

Comment:
Is this working for page blocks or do we need changes there, too?

Reply:
We currently do not have functionality to reset page blocks' design settings